### ISSUE TITLE
AWS TemplateUrl parameters in each template need to be normalized for publication

### ISSUE THREAD

**Issue by robe070** [robe070/cookbooks#109]
The TemplateUrl parameter currently has a complex algorithm to derive the url of the nested template. This does not work for the publishing process. It must be a literal URL.

Instead, PROGRAMMATICALLY, in the pipeline itself after validating it works unmodified, replace the TemplateUrl algorithm with a literal URL. This needs to be done prior to uploading to the support/scalable S3 folder.
All the templates will be here: https://lansa.s3-ap-southeast-2.amazonaws.com/templates/support/scalable.
So all the TemplateUrl parameters will start with that, followed by the filename.
For vpcstack
`"TemplateURL" : "https://lansa.s3-ap-southeast-2.amazonaws.com/templates/support/scalable/nested-vpc.cfn.template"`
For DBWebServerApp & WebServerApp
`"TemplateURL" : "https://lansa.s3-ap-southeast-2.amazonaws.com/templates/support/scalable/webserver-win.cfn.template"`
For MasterStackApp
`"TemplateURL" : "https://lansa.s3-ap-southeast-2.amazonaws.com/templates/support/scalable/lansa-master-win.cfn.template"`
This needs to be done in the 2 parent templates: master and stack type

Write a PowerShell script to do this on the pipeline template artifacts.
The script must be committed to the appropriate LANSA git repo.
The URL values must be constructed from global variables, which are passed into the script as parameters, not entered literally in the script, nor entered literally in the pipeline task:
```
TemplateURLBase = https://lansa.s3-ap-southeast-2.amazonaws.com/templates
TemplateSubFolder = support/scalable
TemplateVPCName = nested-vpc.cfn.template
TemplateWebserverName = webserver-win.cfn.template
TemplateMasterName = lansa-master-win.cfn.template
```

#### When to do this in the pipeline?
There are 2 changes that need to be taken into account. Firstly, by using a literal url, the template location is no longer tied to the cookbooks repo branch name. So the templates can be put anywhere.
Secondly, the production templates are NOT used from this location. AMP upload the templates to their own S3 folders and then modify all the TemplateUrl parameters to point to this S3 folder. So by putting the templates in the support/scalable folder we are not overwriting production templates. The support/scalable folder becomes the location of the templates that will be submitted to preview.

So the appropriate place to do this is a new task before the `upload patch/paas` task - `Normalize TemplateUrl`. And then change  the `upload patch/paas` task to  `upload support/scalable`.

`Normalize TemplateUrl` will run the Powershell script described above.

**Comment by robe070** [comment]
This was implemented as a manual change in the `support/scalable_marketplace` branch
